src/styles/index.scss

```diff
@@ -1,3 +1,148 @@
+:root {
+  --button-color: #00ACDC;
+  --background-color-main: #fff;
+  --price-color: #616070;
+  --cost-color: #060B35;
+}
+
+* {
+  font-family: Roboto, sans-serif;
+}
+
 body {
+  background-color: bisque;
+  width: 100vw;
+  height: 100vw;
+}
+
+.card {
+  width: 200px;
+  height: min-content;
+  box-sizing: border-box;
+  display: block;
+  margin: 0 auto;
+  background-color: var(--background-color-main);
+  border-radius: 5px;
+  border: 1px solid #F3F3F3;
+  padding-bottom: 16;
+}
+
+.card__img {
+  object-fit: contain;
+  display: block;
+  width: 160px;
+  height: 134px;
+  padding: 32 19 0;
+}
+
+.card__title {
+  color: var(--cost-color);
+  height: 36px;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+
+  margin: 40 16 0;
```

Comment:
you should use some points of measurement for margins

Reply:
I don't understand, can you explain in more detail?